src/main/java/org/folio/consortia/service/impl/SharingSettingServiceImpl.java

```diff
@@ -125,11 +139,16 @@ public SharingSettingDeleteResponse delete(UUID consortiumId, UUID settingId, Sh
     log.info("start:: The Sharing Settings for settingId '{}' and '{}' unique tenant(s) were successfully deleted from the database", sharingSettingRequest.getSettingId(), publicationDeleteRequest.getTenants().size());
 
     // we create PC request with POST and PUT Http method to create settings as a consortia-system-user
+    UUID pcId;
+    SharingSettingDeleteResponse sharingSettingDeleteResponse;
     try (var ignored = new FolioExecutionContextSetter(contextHelper.getSystemUserFolioExecutionContext(folioExecutionContext.getTenantId()))) {
-      UUID pcId = publishRequest(consortiumId, publicationDeleteRequest);
-      return new SharingSettingDeleteResponse()
+      pcId = publishRequest(consortiumId, publicationDeleteRequest);
+      sharingSettingDeleteResponse = new SharingSettingDeleteResponse()
         .pcId(pcId);
     }
+    asyncTaskExecutor.execute(
+      () -> updateSettingsToLocalForFailedTenant(consortiumId, pcId, sharingSettingRequest));
```

Comment:
We should return results to UI synchronously

Reply:
fixed